dist/tools/git/git-cache

```diff
@@ -28,6 +41,17 @@ update() {
     set +ex
 }
 
+is_cached() {
```

Comment:
This is for checking if a repository is cached, not if a commit is in the cache. Two different things.